ethereum/contracts/ERC20.sol

```diff
@@ -119,13 +119,10 @@ contract ERC20 is Context, IERC20, IERC20Metadata {
     /**
      * @dev See {IERC20-allowance}.
      */
-    function allowance(address owner, address spender)
-        public
-        view
-        virtual
-        override
-        returns (uint256)
-    {
+    function allowance(
+        address owner,
+        address spender
+    ) public view virtual override returns (uint256) {
```

Comment:
Style changed with this release: https://github.com/prettier-solidity/prettier-plugin-solidity/releases/tag/v1.0.0-rc.1